avocado/core/job.py

```diff
@@ -525,8 +529,12 @@ def create_test_suite(self):
         This is a public Job API as part of the documented Job phases
         """
         refs = self.config.get('references', [])
+        runner_name = self.config.get('test_runner', 'runner')
         try:
-            self.test_suite = self._make_test_suite(refs)
+            if runner_name == 'nrunner':
```

Comment:
@clebergnu, my only "concern" is here. But maybe I'm missing a few things.

If I decide to create a new runner on the future, do I have to add my runner name inside the job module code? 

(Please ignore my comment if makes nonsense. I have to admit that I'm still trying to understand this new logic. You can explain to me later.)